htdocs/components/EG_10_TaxonSelector.js

```diff
@@ -58,46 +59,56 @@ Ensembl.Panel.TaxonSelector = Ensembl.Panel.extend({
 						return false;
 					}
 				}
-			});  
-			
+			});
+
 			var treeObj = panel.elLk.tree.dynatree("getTree");
-			
+
+			//Hiding checkbox for clades/folder if there is no multiselection
+			if(!panel.multiSelect) {
+				panel.elLk.tree.dynatree("getRoot").visit(function(dtnode){
+					if(dtnode.data.isFolder) {
+						dtnode.data.hideCheckbox = true;
+						dtnode.render();
+					}
+				});
+			}
+
 			if (panel.defaultKeys && panel.defaultKeys.length > 0) {
-  			// set selected nodes      
-   			$.each(panel.defaultKeys, function(index, key) { 
-  				var node = treeObj.getNodeByKey(key);
+  			// set selected nodes
+   			$.each(panel.defaultKeys, function(index, key) {
+					var node = treeObj.getNodeByKey(key);
```

Comment:
because I couldn't find anything to comment and just to show that I am too picky... I see you removed all unwanted spaces/tabs but added one here? :-p

Reply:
Not intentional, VScode did all of this :)